staging/src/k8s.io/client-go/testing/fake.go

```diff
@@ -131,13 +131,15 @@ func (c *Fake) Invokes(action Action, defaultReturnObj runtime.Object) (runtime.
 	c.Lock()
 	defer c.Unlock()
 
-	c.actions = append(c.actions, action.DeepCopy())
+	actionCopy := action.DeepCopy()
+	c.actions = append(c.actions, actionCopy)
```

Comment:
Not sure if we should persist `action` or `actionCopy` here...

Reply:
I think `c.actions = append(c.actions, action.DeepCopy())` is correct... that captures a snapshot of the actions invoked